app/views/public/pages.py

```diff
@@ -10,6 +10,8 @@
 def url_view(url):
     from app import get_locale
     page = DataGetter.get_page_by_url('/' + url, get_locale())
+    if page == None:
+		return render_template('gentelella/errors/404.html'), 404
```

Comment:
No. Don't directly handle the 404 page like this. Just call the `abort` method with the status code. let flask handle it.

Eg.
`abort(404)`